google/calendar/UASCalendar.py

```diff
@@ -1,55 +1,66 @@
-import authenticate
+from authenticate import GoogleAuthentication
 
 class UASCalendar:
-    __calendar_id = 'primary'
-    __service = authenticate.authenticate('calendar', 'v3', 'https://www.googleapis.com/auth/calendar')
-    
-    def create_calendar(self, calendarName):
-        resp = self.__service.calendars().insert(body={'summary': calendarName, 'timeZone' : 'America/New_York'}).execute()
-        return resp
-
-    def list_calendars(self):
-        pageToken = None
-        cals = self.__service.calendarList().list(pageToken=pageToken).execute()
-        print cals
-
-    def create_event(self, event_body, notify=False):
-        event = self.__service.events().insert(calendarId=self.__calendar_id, body=event_body.obj).execute()
-        print event.get('htmlLink')
-        return event
-
-    def delete_event(self, event_id, notify=False):
-        self.__service.events().delete(calendarId=self.__calendar_id, eventId=event_id, sendNotifications=notify)
-
-    def delete_events(self, event_id_list, notify=False):
-        for eventt_id in event_id_list:
-            self.__service.events().delete(calendarId=self.__calendar_id, eventId=event_id, sendNotifications=notify)
-    
-    def list_events(self):
-        pageToken = None
-        resp = self.__service.events().list(calendarId=self.__calendar_id, pageToken=pageToken).execute()
-        for i in resp['items']:
-            if 'end' in i and 'timeZone' in i['end'] and 'description' in i and 'htmlLink' in i:
-                print '%s :::: %s :::: %s :::: %s :::: %s to %s' % (i['end']['timeZone'], i['description'], i['created'], i['htmlLink'], i['start']['dateTime'], i['end']['dateTime'])
-
-
-    def update_event(self, event_id, event_body, notify=False):
-        return self.__service.events().update(calendarId=self.__calendar_id, eventId=event_id, body=event_body, sendNotifications=notify).execute()
+	__calendar_id = 'uas.events.tester@gmail.com'
```

Comment:
I was planning on getting the calendar to use from the Google API.  I think it's possible.